### PR TITLE
OCPBUGS-32490: Removed OCI steps from ABI docs

### PR DESCRIPTION
**Pull request by dfitzmau**
MERGE REVIEWER: This PR removes content that is now placed in the Oracle docs. They released their docs yesterday so the OCI links in my doc are now live.

Version(s):
4.14 to 4.16

Issue:
https://issues.redhat.com/browse/OCPBUGS-32490

Link to docs preview:
[Agent-based Installer](https://74910--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_oci/installing-oci-agent-based-installer)

- [x] SME has approved this change.
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/abi-oci-resources-services.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+* A virtual cloud network (VCN). A compute instance, load balancer, and other resources need this network infrastructure to connect to each other over an internet connection.
+* A load balancer, so that you can provide automated traffic distribution on your VCN.
+* A network security group (NSG) in your VCN. You can use the NSG to establish advanced security rules for your network. You must locate the NSG in your compartment, so that certain groups can access network resources.
+* A dynamic group that hosts compute instances. After you create the dynamic group, you can then create a policy statement that defines rules for your cluster environment. This statement sets the precedent for each compute instance to join your {product-title} cluster as a self-managed node.
+* A policy statement, so that your administrator can grant access to your groups, users, or resources that operate in your network.
+* Three Domain Name System (DNS) records and then add the records to a DNS, so that Oracle’s edge-network can maintain your domain’s DNS queries.
```
Comment (snarayan-redhat) [inline modules/abi-oci-resources-services.adoc:21]: Neil's doc has this info
https://docs.google.com/document/d/1zYiZSydr-RkoX6UyYYdOU2QXE6foKcx0HkQwxObom4w/edit#heading=h.pj4nd423cwuc

File: modules/abi-oci-resources-services.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-agent-based-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="abi-oci-resources-services_{context}"]
+= Creating {oci} infrastructure resources and services
+
+You must create an {oci} environment on your virtual machine (VM) shape. By creating this environment, you can install {product-title} and deploy a cluster on infrastructure that supports a wide range of cloud options and strong security policies. Having prior knowledge of {oci} components can help you with understanding the concept of {oci} resources and how you can configure them to meet your organizational needs.
```
Comment (pawanpinjarkar) [inline modules/abi-oci-resources-services.adoc:9]: Could it be? "deploy a cluster on `an` infrastructure" or "deploy a cluster on `any` infrastructure"

**Review by pawanpinjarkar — COMMENTED** [review]
Looks good /lgtm

File: modules/creating-config-files-cluster-install-oci.adoc [hunk 3]
```diff
@@ -92,16 +95,11 @@ pullSecret: '<pull_secret>' <6>
...
+. Access the custom manifest files that are provided in the link:https://docs.oracle.com/iaas/Content/openshift-on-oci/install-prereq.htm#install-configuration-files["Configuration Files"] section of the _Oracle_ documentation.
 +
-[IMPORTANT]
-====
-You must change the secret `oci-cloud-controller-manager` defined in the link:https://github.com/oracle-quickstart/oci-openshift/blob/main/custom_manifests/manifests/oci-ccm.yml[`oci-ccm.yml`] configuration file to match your organization's region, compartment {ocid}, VCN {ocid}, and the subnet {ocid} from the load balancer.
-====
+.. Change the secret `oci-cloud-controller-manager` defined in the link:https://github.com/oracle-quickstart/oci-openshift/blob/main/custom_manifests/manifests/oci-ccm.yml[`oci-ccm.yml`] configuration file to match your organization's region, compartment {ocid}, VCN {ocid}, and the subnet {ocid} from the load balancer.
```
Comment (mburke5678) [inline modules/creating-config-files-cluster-install-oci.adoc:102]:
```suggestion
.. Change the `oci-cloud-controller-manager` secret that is defined in the link:https://github.com/oracle-quickstart/oci-openshift/blob/main/custom_manifests/manifests/oci-ccm.yml[`oci-ccm.yml`] configuration file to match your organization's region, compartment {ocid}, VCN {ocid}, and the subnet {ocid} from the load balancer.
```

File: installing/installing_oci/installing-oci-agent-based-installer.adoc [hunk 4]
```diff
@@ -62,6 +52,8 @@ include::modules/running-cluster-oci-agent-based.adoc[leveloffset=+1]
 .Additional resources
 
 * xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#recommended-resources-for-topologies[Recommended resources for topologies]
+* link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-agent-about-instance-configurations.htm[Instance Sizing Recommendations for OpenShift on {oci} Nodes (Oracle documentation)]
+* link:https://docs.oracle.com/iaas/Content/openshift-on-oci/openshift-troubleshooting.htm[Troubleshooting OpenShift on {oci} (Oracle documentation)]
```
Comment (mburke5678) [inline installing/installing_oci/installing-oci-agent-based-installer.adoc:56]: Keep consistent.
```suggestion
* link:https://docs.oracle.com/iaas/Content/openshift-on-oci/openshift-troubleshooting.htm[Troubleshooting OpenShift on {oci} in the _Oracle_ documentation.
```

File: modules/abi-oci-resources-services.adoc [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+The `api.+*+` and `api-int.+*+` DNS records relate to control plane machines, so you must ensure that all nodes in your installed {product-title} cluster can access these DNS records.
+====
+
+.Prerequisites
+
+* You configured an {oci} account to host the {product-title} cluster. See link:https://docs.oracle.com/iaas/Content/openshift-on-oci/install-prereq.htm[Prerequisites] in the _Oracle_ documentation.
+
+.Procedure
+
+* Create the required {oci} resources and services. See link:https://docs.oracle.com/iaas/Content/openshift-on-oci/agent-prereq.htm[{oci} Resources Needed for Using the {product-title} Agent-based Installer (Oracle documentation)].
```
Comment (mburke5678) [inline modules/abi-oci-resources-services.adoc:30]:
```suggestion
* Create the required {oci} resources and services. See link:https://docs.oracle.com/iaas/Content/openshift-on-oci/agent-prereq.htm[{oci} Resources Needed for Using the {product-title} Agent-based Installer] in the  _Oracle documentation_.
```

**Comment by mburke5678** [comment]
@dfitzmau A few nits. Similar to the [74877 PR](https://github.com/openshift/openshift-docs/pull/74877) from earlier. Could not check the Oracle links as they are not live yet. Otherwise LGTM

File: modules/abi-oci-resources-services.adoc (hunk 5 above)
Comment (dfitzmau) [reply modules/abi-oci-resources-services.adoc:30]: No change as meets https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute

**Comment by mhanss** [comment]
LGTM

File: installing/installing_oci/installing-oci-agent-based-installer.adoc [hunk 6]
```diff
@@ -62,6 +52,8 @@ include::modules/running-cluster-oci-agent-based.adoc[leveloffset=+1]
 .Additional resources
 
 * xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#recommended-resources-for-topologies[Recommended resources for topologies]
+* link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-agent-about-instance-configurations.htm[Instance Sizing Recommendations for OpenShift on {oci} Nodes (Oracle documentation)]
```
Comment (jldohmann) [inline installing/installing_oci/installing-oci-agent-based-installer.adoc:55]:
```suggestion
* link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-agent-about-instance-configurations.htm[Instance Sizing Recommendations for {product-title} on {oci} Nodes (Oracle documentation)]
```

File: installing/installing_oci/installing-oci-agent-based-installer.adoc (hunk 4 above)
Comment (jldohmann) [inline installing/installing_oci/installing-oci-agent-based-installer.adoc:56]:
```suggestion
* link:https://docs.oracle.com/iaas/Content/openshift-on-oci/openshift-troubleshooting.htm[Troubleshooting {product-title} on {oci} (Oracle documentation)]
```

File: modules/running-cluster-oci-agent-based.adoc [hunk 7]
```diff
@@ -4,39 +4,30 @@
...
-+
-For example, from *Compute* -> *Custom images*, import the agent ISO image to the bucket, and enter values in the following fields:
-+
-* Name: *oci-cluster*
-* Bucket: Select the bucket that contains the agent ISO image
-* Object name: Select the name of the agent ISO
-* Image type: *QCOW2*
-
-. After the image imports, go to the *Edit image capabilities* setting and ensure that only `UEFI_64` is selected for the *Firmware* field.
+. Upload the agent ISO image to Oracle’s default Object Storage bucket and import the agent ISO image as a custom image to this bucket. Ensure you that you configure the custom image to boot in Unified Extensible Firmware Interface (UEFI) mode. For more information, see link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-agent-image-creation.htm[Creating the {product-title} ISO Image (Oracle documentation)].
```
Comment (jldohmann) [inline modules/running-cluster-oci-agent-based.adoc:26]:
```suggestion
. Upload the agent ISO image to Oracle’s default Object Storage bucket and import the agent ISO image as a custom image to this bucket. Ensure you that you configure the custom image to boot in Unified Extensible Firmware Interface (UEFI) mode. For more information, see link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-agent-image-creation.htm[Creating the {product-title} Discovery ISO Image (Oracle documentation)].
```

**Review by jldohmann — COMMENTED** [review]
nice work! a couple of minor nits before merge, also your PR needs a rebase

File: modules/running-cluster-oci-agent-based.adoc (hunk 7 above)
Comment (dfitzmau) [reply modules/running-cluster-oci-agent-based.adoc:26]: The "discovery"  ISO image is affiliatd with the Assisted Installer and not the Agent-based Installer, so I'll drop this update.

Comment (jldohmann) [reply modules/running-cluster-oci-agent-based.adoc:26]: gotcha, sounds good 👍

**Pull request merged**